src/Components/BannerCookies/BannerCookies.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  Wrap,
+  Banner,
+  TitleCookie,
+  ParagraphCookie,
+  BtnCookies,
+  WrapContentCenter,
+  LinkPolicy,
+  Icon
+} from "./style.js";
+
+function BannerCookies({ cookie,setCookie }) {
+  return (
+    <div>
+      {!cookie ? (
```

Comment:
Non è molto chiaro a cosa serva la prop cookie. 
Io farei fare lo show/hide del banner al parent, oppure chiamerei la prop in modo diverso (isBannerVisible)